config/suppressions.xml

```diff
@@ -98,5 +98,8 @@
     <!-- HandlerFactory crosses allowed limit for executable statements -->
     <suppress checks="ExecutableStatementCount" files="HandlerFactory\.java"/>
 
+    <!-- can't split long messages between lines -->
+    <suppress checks="RegexpSinglelineJava" files="OrderedPropertiesCheckTest\.java"/>
```

Comment:
What lines can you not split? There should be no way lines can't be split in java. We don't have this issue in our other tests, even those with long strings.

Reply:
You are right, I changed my test. It is not required any more. I removed it.

Reply:
Done.